live-examples/js-examples/map/map.js

```diff
@@ -0,0 +1,17 @@
+const MAP = new Map();
+
+MAP.set('a', 1);
+MAP.set('b', 2);
+MAP.set('c', 3);
+
+let result = MAP.get('a');
+
+console.log(result);
+// expected output: 1
+
+MAP.set('a', 97);
+
+result = MAP.get('a');
+
+console.log(result);
+// expected output: 97
```

Comment:
This example is long enough to count as a "taller" example (https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING.md#short-or-tall-examples) which especially if we omit the `result` variable gives us enough space to show some extra Map features. Maybe something like:

```js

console.log(MAP.size);
// expected output: 3

MAP.delete('b');

console.log(MAP.size);
// expected output: 2
```

?

Reply:
I like that idea. 5fd3fa7 should accomplish this.